styled/rewards.js

```diff
@@ -1,13 +1,18 @@
 import { Platform, StyleSheet } from 'react-native';
 import styled from 'styled-components/native';
-import { Subhead, Caption } from '../components/BaseComponents';
+import { Subhead, Caption, NavButton } from '../components/BaseComponents';
 import Colors from '../assets/Colors';
 
 export const Container = styled.View`
   flex: 1;
   background-color: #fff;
 `;
 
+export const BackButton = styled(NavButton)`
```

Comment:
It took me a while to realize why this had a green background. I feel like this could get messy setting background colors when they mostly don't have backgrounds...see my comment in BaseComponents about this

Reply:
ALSO then you wouldn't have to extend this! You'd just use props when you need a button with a background.